package.json

```diff
@@ -77,12 +79,18 @@
     "ember-resolver": "^2.0.3",
     "ember-try": "^0.2.5",
     "ember-watson": "^0.7.0",
+    "express": "^4.14.0",
     "github": "^0.2.4",
     "glob": "5.0.13",
+    "heimdalljs": "~0.3.0-alpha2",
+    "heimdall-query": "^0.0.4",
+    "json-api-mock-server": "0.0.2",
     "loader.js": "^4.0.0",
+    "lodash": "^4.15.0",
```

Comment:
I'm not sure lodash is directly needed anymore, I'll do a quick review to make sure. I believe this was only here for what is now heimdall-query.